config/routes.rb

```diff
@@ -5,5 +5,8 @@
   get 'sign_in', to: 'sessions#new'
   post 'sign_in', to: 'sessions#create', as: 'log_in'
   delete 'logout', to: 'sessions#destroy'
+  resources :users do
+    resources :addresses
+  end
```

Comment:
How many routes this logic will generate? And are we using all of the routes?

Reply:
its will generate all routes 
fixed by specifying only required routes